packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -867,6 +867,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
       ...getItemProps({
         item,
         index,
+        disabled: !!item['disabled'],
```

Comment:
```suggestion
        disabled: item['disabled'],
```

Is it really required?

Reply:
double checked in downshift and it's not. will remove, thanks!